### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by udev-21**
Prepared the first two sections (toolbar and headline) of the third template according to requirements.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <header class="toolbar-container">
+        <div class="welcome">
+            <p>Welcome</p>
+        </div>
+        <div class="toggle-wrapper">
+            <div class="icon-wrapper">
+                <i class="fa-solid fa-bars fa-lg"></i>
+            </div>
+        </div>
+    </header>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [x] Good Job for using `header` tag for toolbar section, But I recommend you to use `a` tag for Logo("welcome") and for hamburger menu use a `nav` tag  for semantics reason.
- [x] Good Job for Toolbar section has the proper background image.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <section class="intro">
+        <div class="greeting">
+            <div>Hey there. I’m Sharof</div>
+            <div class="green">I’m a software developer</div>
+        </div>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [x] Good Job for using `section` tag for Headline section, But I recommend you to use `h1` tag for main title("Hey there. I’m Sharof") and for subtitle("I’m a software developer") use a `h2` tag for semantics reason.
- [x] Good Job for Headline section has the proper background image as design in Figma Template.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        </p>
+        <section class="social">
+            <a href="">
+                <i class="fa-brands fa-twitter"></i>
+            </a>
+            <a href="linkedin.com/in/udev21"><i class="fa-brands fa-linkedin fa-xl"></i></a>
+            <a href="github.com/udev-21"><i class="fa-brands fa-github fa-xl"></i></a>
+            <a href="t.me/udev21"><i class="fa-brands fa-telegram fa-xl"></i></a>
+            <a href="#"><i class="fa-brands fa-angellist fa-xl"></i></a>
+        </section>
```
Comment (Abenezer-Tilahun) [inline index.html]: - [x] Kindly, For social icons  uses a `ul` tag to group them, and `li` per icon; each icon has a link `a` tag to  let you connect your social networks with the icons, using `href.`
- [x] Kindly, try to stick to the design as much as possible ( For social icons there is space between icons) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
For social icons your project look likes : 


The Figma Design Look likes : 

f3

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @udev-21  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Good commit messages.

almost

## Required Changes ♻️
_Check the comments below the review._

- [x] Pull Request should have a descriptive title telling you what changes are inside and should have a summary describing introduced changes in more detail.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @udev-21, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by udev-21** [comment]
I'm so grateful.
Thank you so much 👍🏽 
@mirouhml

**Pull request merged**